From 7a6292d481fbb8f7c9b1d511e6679c26605b9e00 Mon Sep 17 00:00:00 2001
From: lizhi2 <lizhi2@eswincomputing.com>
Date: Mon, 25 Aug 2025 09:35:04 +0800
Subject: [PATCH 470/533] feat(eth): update 1000M dly of Z530 end0

Changelogs:
1. update 1000M dly data of Z530 end0 based on the board produced in batches at the factory.

Signed-off-by: lizhi2 <lizhi2@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/eic7700-z530.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-z530.dts b/arch/riscv/boot/dts/eswin/eic7700-z530.dts
index a790850fda0e..3db40fb3cecf 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-z530.dts
+++ b/arch/riscv/boot/dts/eswin/eic7700-z530.dts
@@ -45,7 +45,7 @@ reg_vcc1v8: vcc1v8 {
 };
 
 &d0_gmac0 {
-	dly-param-1000m = <0x10101010 0x6E103210 0x10101010>;
+	dly-param-1000m = <0x10101010 0x1D003410 0x0>;
 	dly-param-100m = <0x10101010 0x64104610 0x10101010>;
 	dly-param-10m = <0 0 0>;
 };
-- 
2.53.0

